exporter/index.js

```diff
@@ -22,29 +22,34 @@ const validation = {
 };
 
 let sentryClient;
-const sentryIsEnabled = () => sentryClient == null;
+const sentryIsEnabled = () => !(sentryClient == null);
 
-if (process.env.SENTRY_DSN == null && process.env.SENTRY_RELEASE == null
-  && process.env.SENTRY_ENVIRONMENT == null && process.env.SENTRY_SERVER_NAME == null) {
-  console.log('Sentry not configured');
-} else {
+const enableSentry = !(
```

Comment:
not sure this logic seems correct. should all of them be set or at least one? I think all is correct

Reply:
All of them should be set. Have I fooled myself?

Reply:
I think if one is not equal to `null` then `enableSentry` will be `true`. Have _i_ fooled myself?

Reply:
Yes, I think you're right. I was too focused trying to support both null and undefined & fooled myself.